src/components/DataTable/DataTable.tsx

```diff
@@ -36,6 +36,10 @@ export interface DataTableProps {
   showTotalsInFooter?: boolean;
   /** Lists of data points which map to table body rows. */
   rows: TableData[][];
+  /** Show a <Navigation /> when the data table is horizontally scrollable.
```

Comment:
Let me know if there's a proper way to word this

Reply:
Maybe without the angle brackets.